Dalamud/Game/Config/UiConfigOption.cs

```diff
@@ -3314,4 +3314,165 @@ public enum UiConfigOption
+    NamePlateEdgeHealer,
+
+    /// <summary>
+    /// System option with the internal name NamePlateColorDps.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("NamePlateColorDps", ConfigType.UInt)]
+    NamePlateColorDps,
+
+    /// <summary>
+    /// System option with the internal name NamePlateEdgeDps.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("NamePlateEdgeDps", ConfigType.UInt)]
+    NamePlateEdgeDps,
+
+    /// <summary>
+    /// System option with the internal name NamePlateColorOtherClass.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("NamePlateColorOtherClass", ConfigType.UInt)]
+    NamePlateColorOtherClass,
+
+    /// <summary>
+    /// System option with the internal name NamePlateEdgeOtherClass.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("NamePlateEdgeOtherClass", ConfigType.UInt)]
+    NamePlateEdgeOtherClass,
+
+    /// <summary>
+    /// System option with the internal name NamePlateDispWorldTravel.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("NamePlateDispWorldTravel", ConfigType.UInt)]
+    NamePlateDispWorldTravel,
+
+    /// <summary>
+    /// System option with the internal name LogNameIconType.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("LogNameIconType", ConfigType.UInt)]
+    LogNameIconType,
+
+    /// <summary>
+    /// System option with the internal name LogDispClassJobName.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("LogDispClassJobName", ConfigType.UInt)]
+    LogDispClassJobName,
+
+    /// <summary>
+    /// System option with the internal name LogSetRoleColor.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("LogSetRoleColor", ConfigType.UInt)]
+    LogSetRoleColor,
+
+    /// <summary>
+    /// System option with the internal name LogColorRoleTank.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("LogColorRoleTank", ConfigType.UInt)]
+    LogColorRoleTank,
+
+    /// <summary>
+    /// System option with the internal name LogColorRoleHealer.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("LogColorRoleHealer", ConfigType.UInt)]
+    LogColorRoleHealer,
+
+    /// <summary>
+    /// System option with the internal name LogColorRoleDPS.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("LogColorRoleDPS", ConfigType.UInt)]
+    LogColorRoleDPS,
+
+    /// <summary>
+    /// System option with the internal name LogColorOtherClass.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("LogColorOtherClass", ConfigType.UInt)]
+    LogColorOtherClass,
+
+    /// <summary>
+    /// System option with the internal name ItemInventryStoreEnd.
+    /// This option is a UInt.
+    /// </summary>
+    [GameConfigOption("ItemInventryStoreEnd", ConfigType.UInt)]
+    ItemInventryStoreEnd,
```

Comment:
Is this actually SE spelling, or is it a CS typo we want to fix before exposing new public API?

Reply:
It's all directly from the game in the way it is required to be.

Reply:
It's SE:

![image](https://github.com/goatcorp/Dalamud/assets/5192145/7f37441a-62cc-4493-8a9e-00a18b47580a)